README.md

```diff
@@ -1 +1,153 @@
-# redux-group-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Space Travelers Hub README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Space Travelers Hub ] <a name="about-project"></a>
+
+**[Space Travelers Hub ]** is a website that display a list of space rockets and missions, and how to reserve them.
+
+## 🛠 Built With <a name="built-with">React.js</a>
+
+### Tech Stack <a name="tech-stack">CSS</a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">REACT JS </a></li>
+
+    <li><a href="#">JSX</a></li>
+     <li><a href="#">Js</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Display a list of Rockets and how to reserve them]**
+- **[Display a couple of missions for each rocket and how to reserve them]**
+- **[Displaying the profile section that shows whether you have reserved a rocket or mission]**
+  **[State is managed with Redux]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[N/A]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- In order to run this project you need:
+
+- [git](https://git-scm.com/downloads): A tool for managing source code
+- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
+- [Node.js](https://nodejs.org/en): Node.js is an open-source, cross-platform JavaScript runtime environment.
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/Otegaa/redux-group-project.git
+Example commands:
+
+```sh
+  cd redux-group-project
+  https://github.com/Otegaa/redux-group-project.git
+```
+
+### Install
+
+Install this project with:
+
+- Run npm install after cloning the repo
+
+### Usage
+
+### Usage
+
+- Run npm start for development.
+- This runs the application in the development mode
+- Open http://localhost:3000/ to view it in the browser.
+- Run npm run build for production
+
+### Run tests
+
+To run tests, run the following command:
+` npm run test`
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@ridabensalem](https://github.com/ridabnesalem)
```

Comment:
Great job team🤝

- [ ] Kindly ensure the GitHub social handle link is not broken. Fixing this will make your readme file more professional.